pkg/snapshot/snapshotcontroller.go

```diff
@@ -69,11 +75,33 @@ func (s *SnapshotController) Start() error {
 		return err
 	}
 
+	log.Infof("Registering CRDs")
 	err = client.CreateCRD(aeclientset)
 	if err != nil {
 		return err
 	}
 
+	storkRuleResource := k8s.CustomResource{
+		Name:    "storkrule",
```

Comment:
As a separate PR can you update the CRD names to not have stork in the name. They are already in the stork group, the names for the types don't need to have it.

Reply:
Sure, after this is merged

Reply:
https://github.com/libopenstorage/stork/issues/128 for this